variables.md

```diff
@@ -27,6 +27,7 @@ CHECK_RELEASENOTES_ORIGIN | boolean | false | Loads `installation/releasenotes_o
 CHECKSUM_* | string | | SHA256 checksum of the * medium. E.g. CHECKSUM_ISO_1 for ISO_1.
 CHECKSUM_FAILED | string | | Variable is set if checksum of installation medium fails to visualize error in the test module and not just put this information in the autoinst log file.
 DESKTOP | string | | Indicates expected DM, e.g. `gnome`, `kde`, `textmode`, `xfce`, `lxde`. Does NOT prescribe installation mode. Installation is controlled by `VIDEOMODE` setting
+DEPENDENCY_RESOLVER_FLAG| boolean | undef      | Control when the resolve_dependecy_issues will be scheduled or not.
```

Comment:
It needs some rephrase now this description and default we can say that is false.